inbox/headless/api-reference.mdx

```diff
@@ -0,0 +1,331 @@
+---
```

Comment:
We should somewhere in the docs say like: if your application UI library/framework is not supported yet you can use our headless js sdk to build your custom UI; and link to the headless page.